script.js

```diff
@@ -19,39 +19,42 @@ recipeApp.getRecipe = () => {
             }
         }).then((res) => {
             for (let i = 0; i < 3; i++) {
-                console.log("this is length of needed array", res.meals.length);
+                // get three random dishes and limiting output on page set to only 3
                 const random = [Math.floor(Math.random() * res.meals.length)];
 
                 const title = res.meals[random].strMeal;
                 const dishImage = res.meals[random].strMealThumb;
                 const recipeId = res.meals[random].idMeal;
 
 
-                // second call to get recipe by id
+                // second call to get recipe and finding details by id
+
                 $.ajax({
-                    url: `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${recipeId}`,
-                    method: 'GET',
-                    datatype: 'json',
-                    data: {
-                        per_page: '3',
-                    }
-                }).then((res) => {
-                    console.log(res, "second call warks")
-                    const recipe = res.meals[0].strInstructions;
-                    const ingredient = res.meals[0].strIngredient1
-                    console.log(ingredient)
-                    console.log("here is an output of", recipe);
+                        url: `https://www.themealdb.com/api/json/v1/1/lookup.php?i=${recipeId}`,
+                        method: 'GET',
+                        datatype: 'json',
+                        data: {
+                            per_page: '3',
+                        }
+                    })
+                    .then((res) => {
+                        // renedering title, id, recipe and image of selected dishes
+                        console.log(res, "second call warks")
+                        const recipe = res.meals[0].strInstructions;
+                        const ingredient = res.meals[0].strIngredient1
+                        console.log(ingredient)
+                        console.log("here is an output of", recipe);
 
-                    $('body').append(`

```

Comment:
here, we better add a div inside of body, because if we want to empty the page, we can only empty the body instead of div, then everything will be gone including buttons.